be/docker/be-dev.dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:lts-alpine
+
+WORKDIR /app
+
+# append node_modules to PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# copy project files (docker-compose seems ignore '.dockerignore' files.
+# If you can make docker-compose ignore folders such as 'node_modules' please
+# make a pull request doing that (and replace these copy commands with 'COPY . ./').)
+COPY .env.example ./
+COPY .env ./
+COPY .eslintrc.js ./
+COPY .prettierrc ./
+COPY nest-cli.json ./
+COPY package-lock.json ./
```

Comment:
I mean at this point wouldn't it be easier to just copy all and then delete some?

Reply:
Also this just copys them at build so if you need to change anything you will need to rebuild docker image. Use docker-compose.yml volumes instead. just mount it to correct place and all good.

Reply:
you are kinda mixing up dev docker and production docker and ci docker here I am feeling like.

Reply:
Yep this was supposed to be a dev-docker setup task (I don't know who would use it though as all the things run quite well locally other than the db)

Reply:
> Also this just copys them at build so if you need to change anything you will need to rebuild docker image. Use docker-compose.yml volumes instead. just mount it to correct place and all good.

The thing about docker volumes is that they don't want to work with existing folders (for me) and newly created volumes have a different owner and permissions, which means that reading and writing to those folders becomes a pain (on Linux). Maybe i'm doing it wrong (if so please let me know).

Any new ideas are welcome.

Reply:
Fixed with commit 16510fb986410b1627482c137abd79ed92f2e5b0.